### PR TITLE
Add grafana-plugin-cli

### PR DESCRIPTION
**Pull request by corpglory-dev**
Add [grafana-plugin-cli](https://github.com/CorpGlory/grafana-plugin-cli) to `packages/`

### CODE REVIEW

File: packages/grafana-plugin-cli/.babelrc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "presets": [
+    ["env", {
+      "targets": {
+        "node": "6.14"
```
Comment (jonyrock) [inline packages/grafana-plugin-cli/.babelrc]: I thing we can use this target somewhere from Grafana's target
but we can keep it here

File: packages/grafana-plugin-cli/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# Grafana-plugin-cli
+
+Create Grafana plugins that suit your needs.
+
+No build configuration needed. 
+Just run CLI commands and choose whether your plugin will be:
+- Panel / Datasource
+- TypeScript / JavaScript
```
Comment (jonyrock) [inline packages/grafana-plugin-cli/README.md]: Please remove what it doesn't do

File: packages/grafana-plugin-cli/README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+```
+npm install
+npm run build
+npm start
+```
+
+## Options description
+```
+-----------------------------
+Welcome to Grafana-plugin-cli
```
Comment (jonyrock) [inline packages/grafana-plugin-cli/README.md]: lets also clean this

File: packages/grafana-plugin-cli/README.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# Grafana-plugin-cli
+
+Template generators based on `ejs` (https://github.com/mde/ejs
```
Comment (rozetko) [inline packages/grafana-plugin-cli/README.md]:
```suggestion
Template generators based on [ejs](https://github.com/mde/ejs)
```

**Pull request closed without merging**